wordpress-mobile/WordPress-iOS
# Update status for phantom uploading media.

Fixes #8310 

This PR by checking the status of Media when the app is started from scratch.
All media that is in the ```.pushing state``` is updated to ```.failed```.

Note that this behaviour will need to be changed when we start to do full async uploads, at that time we first need to check if there is no pending background task still going for that upload before deciding to change the status.

To test:
 - Start the app
 - Go to the media library
 - Add an media object and wait for start of upload of media
 - Kill the app (stop in Xcode)
 - Restart the app
 - Go to the same media library
 - See the status of media that was cancelled
 - You should be able to retry that media.
